server/src/routes/api/v1/users/index.ts

```diff
@@ -0,0 +1,32 @@
+import { Router, Request, Response, NextFunction } from "express";
+import { UserService } from "./../../../../logic/services/user.service";
+import { SocoboUser } from "./../../../../models/socobouser";
+
+export class UsersRouteV1 {
+  private _userService: UserService;
+  private _router: Router; 
+
+  constructor (userService: UserService, router: Router) {
+    this._userService = userService;
+    this._router = router;
+  }
+
+  createRoutes (): Router {
+    // get all users
+    this._router.get("/", (req: Request, res: Response, next: NextFunction) => {
+      this._userService.getAllUsers()
+        .then((result: SocoboUser[]) => res.status(200).json(result))
```

Comment:
I Just called them magic Numbers because they are hard coded. Ok maybe this is not completly true because status codes have actually a meaning. But I think named variables would be slightly more readable. Because I not always know what the status code means if i only see the number